google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/BreakpointUtil.java

```diff
@@ -84,11 +86,17 @@ public static Date parseDateTime(@Nullable String dateString) {
     dateString = dateString.replaceAll("Z$", "-0000");
 
     SimpleDateFormat iso8601Format = new SimpleDateFormat(ISO_8601_FORMAT);
+    iso8601Format.setLenient(true);
 
     try {
```

Comment:
I would prefer your way if we had more than 2 formats. Realistically, we'll have two for a short time and then back to one. Which reminds me, I should also add a TODO to check on the no_ms rollout status and eventually clean up.